queue/variables.tf

```diff
@@ -38,10 +38,36 @@ variable "max_receive_count" {
 }
 
 variable "alarm_topic_arn" {
-  description = "ARN of the topic where to send notification for DLQs not being empty. If null, no alarm will be created."
+  description = "DEPRECATED, use dlq_alarm_topic_arn: ARN of the topic where to send notification for DLQs not being empty. If null, no alarm will be created."
   default     = null
 }
 
+variable "enable_dlq_not_empty_alarm" {
+  description = "DEPRECATED, use dlq_alarm_topic_arn: Enable alarm for DLQ not being empty"
```

Comment:
Should this be deprecated? 
We have "enable_queue_age_alarm" and "main_q_age_alarm_action_arns" for old messages alarms
Would make sense to have "enable_dlq_not_empty_alarm" and "dlq_alarm_action_arns" for DLQ alarms for consistency?

Reply:
There will be a possibility of enabling the alarms but no action to be taken if "enable_alarms" variable is `true` but "alarm_action_arns" is `[]`. Ok if we want that, otherwise, only "enable_alarms" if "alarm_action_arns" != []

Reply:
> Should this be deprecated?

No, i'll update the text.

> Would make sense to have "enable_dlq_not_empty_alarm" and "dlq_alarm_action_arns" for DLQ alarms for consistency?

Yes, these have been added already a bit further down.

> Ok if we want that, otherwise, only "enable_alarms" if "alarm_action_arns" != []

Alarms can be useful even without configured actions, and still can trigger actions via EventBridge rules even if no action is configured. I've kept the ability to enable alarms with no actions to accommodate that.